account_invoice_date_maturity/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Account Invoice Date Maturity",
+    "summary": """
+        Adds date_maturity field with Journal Entries action
```

Comment:
Better this (human readable field name, and right model name):
```suggestion
        Makes Due Date visible for Journal Items
```